airflow-core/src/airflow/utils/process_utils.py

```diff
@@ -184,7 +196,18 @@ def execute_in_subprocess_with_kwargs(cmd: list[str], **kwargs) -> None:
     :param cmd: command and arguments to run
 
     All other keyword args will be passed directly to subprocess.Popen
+
+    .. deprecated:: 3.2.0
+        This function is deprecated. Please implement your own subprocess execution logic,
+        reference the one present in standard / google providers.
     """
+    import warnings
+
+    warnings.warn(
```

Comment:
Nit: this will warn twice when `execute_in_subprocess` is called. Not sure it's worth caring about though.